pyiron_base/database/generic.py

```diff
@@ -880,10 +900,10 @@ def get_items_dict(
                 self.conn.connection.create_function("like", 2, self.regexp)
 
             result = self.conn.execute(query)
-        row = result.fetchall()
+        results = [row._asdict() for row in result.fetchall()]
```

Comment:
A slightly faster (~10%) alternative would be
```suggestion
        results = result.mappings().all()
```
but the result is then a list of sqlalchemy objects that behave like dicts, not standard dicts.  One can also feed `result.mappings()` directly into a DataFrame directly, but that would change api more dramatically.

Reply:
Just to add some numbers for a query on a project with ~2M jobs:
1. ~10s for the plain sqlalchemy query
2. ~20s for a call to get_items_dict
3. ~30s for a call to job_table

Reply:
database reports around 3s for your `2024%` queries. The rest seems to be sqlalchemy and pyiron.